index.html

```diff
@@ -522,20 +536,25 @@ <h4>Script Type</h4>
                 <p><a>Script Events</a> in this type of <a>script</a> SHOULD contain <a>Text</a> objects
```

Comment:
Also wondering why we have a `SHOULD` and not a `SHALL`. I would suggest closing https://github.com/w3c/dapt/issues/52 by saying that representing unfinished documents is out of scope, and using the SHALL here.

Reply:
My instinct is that I'd rather not do that, and that it'd be somewhat unfriendly, but I'd prefer to get implementation and usage feedback before deciding to do that.